Source/movie.cpp

```diff
@@ -60,9 +60,11 @@ void play_movie(const char *pszMovie, bool userCanClose)
 #ifndef USE_SDL1
 				case SDL_WINDOWEVENT:
 					if (event.window.event == SDL_WINDOWEVENT_FOCUS_LOST)
-						diablo_focus_pause();
-					else if (event.window.event == SDL_WINDOWEVENT_FOCUS_GAINED)
-						diablo_focus_unpause();
+						if (*sgOptions.Gameplay.pauseOnFocusLoss)
+							diablo_focus_pause();
+						else if (event.window.event == SDL_WINDOWEVENT_FOCUS_GAINED)

```

Comment:
`if ... else` is considered a single statement, the else condition is being evaluated as part of the immediately preceding if statement and will never be true (since you're already inside an `if (event == FOCUS_LOST)` block. You need to use braces here.

Reply:
Whoops, reformatted and fixed